### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by drblobfish** [mahendra-mariadassou/learningr#3]
`characters` doesn't make sense. I think the correct word is `vectors`.
If the correct word is indeed `character`, it would be nice to explain the terminology

https://github.com/mahendra-mariadassou/learningr/blob/ec554424dbdaa277acef637d6c80a4ca773f1eb5/inst/tutorials/01_programming_basics/datatypes.Rmd#L283

**Comment by mahendra-mariadassou** [comment]
Indeed, many thanks for noticing the typo. I'll correct it and you'll have to update the package with 
```
remotes::install_github("mahendra-mariadassou/learningr")
```
and relaunch the tutorial for it to be corrected on your computer. Please start a new R session before updating the package and relaunching the tutorial (to avoid any conflicts when building tutorials).
